src/bundle/Resources/views/themes/admin/ui/dashboard/block/all.html.twig

```diff
@@ -1,8 +1,8 @@
 {% trans_default_domain 'dashboard' %}
 
-<div class="card border-light mb-4">
+<div class="card border-light ez-card ez-card--no-padding mb-4">
+    <div class="ez-card__title">{{ 'everyone'|trans|desc('Common content') }}</div>
     <div class="card-body">
-        <h2 class="mb-3">{{ 'everyone'|trans|desc('Common content') }}</h2>
         {{ ez_render_component_group('dashboard-all-tab-groups') }}
```

Comment:
Is this perhaps slightly wrong if we look to how bootstrap is organizing titles in cards?
https://getbootstrap.com/docs/4.5/components/card/#titles-text-and-links

This is defiantly nitpick, but asking as we where using this as an example in training on how to customize Admin UI.